geoips/interfaces/base.py

```diff
@@ -98,17 +97,12 @@ def plugin_module_to_obj(interface, module, module_call_func="call", obj_attrs={
     try:
         obj_attrs["__doc__"] = module.__doc__
     except AttributeError:
-        if plugin_deprecations:
-            warn(
-                f"Plugin module '{module.__name__}' does not have a docstring. "
-                "Please add a module-level docstring",
-                DeprecationWarning,
-                stacklevel=1,
-            )
-        else:
-            raise PluginError(
-                f"Required '__doc__' attribute not found in '{module.__name__}'"
-            )
+        raise PluginError(
+            f"Required '__doc__' attribute not found in '{module.__name__}'"
+        )
+    # Don't know if we want to add this explicit "docstring" attribute - if we do,
+    # module-based plugins will have EXACTLY the same attributes as yaml-based plugins.
+    obj_attrs["docstring"] = module.__doc__
```

Comment:
Note I have added both the __doc__ and "docstring" attributes, so we match YAML plugins better.  Do you think we want both on module-based plugins?

Reply:
I think this is a good idea, at least for now. We can see how it works in practice. Sometimes I don't like having the same information duplicated in multiple attributes, but this is fine for now.